react-native/architecture/components/my-component/style.md

```diff
@@ -0,0 +1,8 @@
+# [Standard] Components folder - Component style file
+
+## Standard
+
+- Has a `export default` of a `Stylesheet.create`
+- The file name should be `MyComponent.style.js`
```

Comment:
Spliting style was done on "Ch..z" and that lead, to me, to less readable files.

I disagree with xxxx.style.js files and I would prefer to push the styled component approach.
```js
import styled from 'styled-components/native';

const StyledView = styled.View`
  background-color: papayawhip;
`;

const StyledText = styled.Text`
  color: palevioletred;
`;

class MyReactNativeComponent extends React.Component {
  render() {
    return (
      <StyledView>
        <StyledText>Hello World!</StyledText>
      </StyledView>
    )
  }
}
```